e3sm_to_cmip/util.py

```diff
@@ -73,227 +113,240 @@ def setup_cmor(var_name, table_path, table_name, user_input_path):
-            if handler['name'] not in table_info['variable_entry']:
+            table_info = _get_table_info(tables, handler["table"])
+            if handler["name"] not in table_info["variable_entry"]:
                 msg = f"Variable {handler['name']} is not included in the table {handler['table']}"
                 print_message(msg, status="error")
                 continue
             else:
                 msg = {
-                    "CMIP6 Name": handler['name'],
-                    "CMIP6 Table": handler['table'],
-                    "CMIP6 Units": handler['units'],
-                    "E3SM Variables":  ', '.join(handler['raw_variables'])
+                    "CMIP6 Name": handler["name"],
+                    "CMIP6 Table": handler["table"],
+                    "CMIP6 Units": handler["units"],
+                    "E3SM Variables": ", ".join(handler["raw_variables"]),
                 }
-                if handler.get('unit_conversion'):
-                    msg["Unit conversion"] = handler['unit_conversion']
-                if handler.get('levels'):
-                    msg["Levels"] = handler['levels']
+                if handler.get("unit_conversion"):
+                    msg["Unit conversion"] = handler["unit_conversion"]
+                if handler.get("levels"):
+                    msg["Levels"] = handler["levels"]
                 messages.append(msg)
-    
+
     elif freq and tables and inpath:
-        file_path = next(Path(inpath).glob('*.nc'))
-        
-        resource_path, _ = os.path.split(os.path.abspath(resources.__file__))
-        defaults_path = os.path.join(
-            resource_path,
-            'default_handler_info.yaml')
-        
-        with open(defaults_path, 'r') as infile:
-            default_info = yaml.load(infile, Loader=yaml.SafeLoader)
-        
-        
+        file_path = next(Path(inpath).glob("*.nc"))
 
         with xr.open_dataset(file_path) as ds:
             for handler in handlers:
-                
-                table_info = get_table_info(tables, handler['table'])
-                if handler['name'] not in table_info['variable_entry']:
+
+                table_info = _get_table_info(tables, handler["table"])
+                if handler["name"] not in table_info["variable_entry"]:
                     continue
 
                 msg = None
                 raw_vars = []
-                for default in default_info:
-                    if handler['name'] + "_highfreq" == default['cmip_name'] and freq in default['table']:
-                        msg = {
-                            "CMIP6 Name": default['cmip_name'],
-                            "CMIP6 Table": default['table'],
-                            "CMIP6 Units": default['units'],
-                            "E3SM Variables":  [default['e3sm_name']]
-                        }
-                        raw_vars.append(default['e3sm_name'])
-                        break
```

Comment:
This basically replaced the `"_highfreq"` portion of the variable name (passed by the user in var list) with the CMIP name. It is no longer required because the user passes the CMIP legal name without `"_highfreq"` now.

I don't think this check even worked because `pr_highfreq` and `rlut_highfreq` are Python modules and not defined as a default handler in `default_handler_info.yaml`.

Reply:
Excellent!  Then I no longer need these codes in "GenerateAtmDayCMIP.py" and "GenerateAtm3hrCMIP.py":

>         list1 = [ f"{x}_highfreq" for x in cmip_var if x in ['pr', 'rlut'] ]
>         list2 = [ x for x in  cmip_var if x not in ['pr', 'rlut'] ]
>         cmip_vars = list1 + list2